nuntium/models.py

```diff
@@ -727,16 +746,21 @@ def send_an_email_to_the_author(sender, instance, created, **kwargs):
         if settings.SEND_ALL_EMAILS_FROM_DEFAULT_FROM_EMAIL:
             from_email = settings.DEFAULT_FROM_EMAIL
         else:
+            from_domain = settings.DEFAULT_FROM_DOMAIN
```

Comment:
You could write this as an or, using the short-circuit:

`from_domain = confirmation.message.writeitinstance.config.custom_from_domain or settings.DEFAULT_FROM_DOMAIN`